docs/getting-started/advanced-topics/env-configuration.md

```diff
@@ -291,6 +291,10 @@ More information about this setting can be found [here](https://docs.sqlalchemy.
 - Default: `8080`
 - Description: Sets the port to run Open WebUI from.
 
+:::info
+If installed via Python, you must instead pass `--port` as a command line argument.
```

Comment:
It's not a "requirement", you might want to update the wording for a better clarity here!

Reply:
I do believe it is a requirement. 

If you run `open-webui serve`, setting `PORT` to something other than the default, it will not listen on that port. You must pass the port like `open-webui serve --port 8001`.

FWIW, we installed via Conda. Perhaps if installed via pip(venv) it may not have this behavior, but I would expect it would behave the same.